### DISCUSSION TITLE
Hot End Temps

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#1090] [General]
Miguel, all,
As I have an all metal hot-end on my E3-V2, there have been two times I thought I might print at higher temps than 260c (the current limit permitted by the firmware) but can't.  I'm on the MSROC firmware 2.13. 

Is there a line in the Marlin Firmware G-Code that I can use to address this?  Is it as simple as changing this line in Marlin?
#define HEATER_0_MAXTEMP 275

Hate to be the ignorant one but I am holding my hand up...I'm that person.
Thanks in advance, PDC

**Comment by mriscoc** [comment]
Look for C104 custom G-code, there is a video in the videos section of the Wiki. In short you need to write something like:
```gcode
C104 U1 T315
```
For reach a working temperature of 300°C but be careful.

**Comment by Pridanc** [comment]
Miguel, 
Many thanks for the information.  Also thanks for the warning to be careful. As an ex car racer I can say I'm very careful!  Stay well, PDC
